tests/pytests/unit/modules/test_aixpkg.py

```diff
@@ -0,0 +1,1545 @@
+import logging
+import os
+
+## import salt.modules.rpm_lowpkg as rpm
+## import salt.modules.yumpkg as yumpkg
+import salt.modules.aixpkg as aixpkg
+
+## import salt.modules.cmdmod as cmdmod
```

Comment:
There quit a few imports commented out in this test file. Can you remove?

Reply:
cleaned up